Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -4667,6 +4688,488 @@ def apply_quarantine_file_action_command(args: dict) -> CommandResults:
+        'description': args.get('description'),
+        'quarantine': argToBoolean(args.get('quarantine')) if args.get('quarantine') is not None else None,
+        'pause_duration': arg_to_number(args.get('pause_duration')),
+        'sensor_ml_level_detection': arg_to_number(args.get('sensor_ml_level_detection')),
+        'sensor_ml_level_prevention': arg_to_number(args.get('sensor_ml_level_prevention')),
+        'cloud_ml_level_detection': arg_to_number(args.get('cloud_ml_level_detection')),
+        'cloud_ml_level_prevention': arg_to_number(args.get('cloud_ml_level_prevention')),
+        'max_duration': arg_to_number(args.get('max_duration')),
+    }
+
+    if is_scheduled:
+        result['schedule'] = {
+            'interval': SCHEDULE_INTERVAL_STR_TO_INT.get(args['schedule_interval'].lower()),
+            'start_timestamp': (
+                dateparser.parse(args['schedule_start_timestamp'])
+                or return_error('Invalid start_timestamp.')
+            ).strftime("%Y-%m-%dT%H:%M"),
+        }
+
+    else:
+        result['hosts'] = argToList(args.get('hosts'))
+
+    return result
+
+
+def ODS_create_scan_request(args: dict, is_scheduled: bool) -> dict:
+    body = make_create_scan_request_body(args, is_scheduled)
+    remove_nulls_from_dictionary(body)
+    return http_request('POST', f'/ods/entities/{"scheduled-"*is_scheduled}scans/v1', json=body)
+
+
+def ODS_verify_create_scan_command(args: dict) -> None:
+
+    if not (args.get('hosts') or args.get('host_groups')):
+        raise DemistoException('MUST set either hosts OR host_groups.')
+
+    if not (args.get('file_paths') or args.get('scan_inclusions')):
+        raise DemistoException('MUST set either file_paths OR scan_inclusions.')
+
+
+def ods_create_scan(args: dict, is_scheduled: bool) -> dict:
+
+    ODS_verify_create_scan_command(args)
+
+    response = ODS_create_scan_request(args, is_scheduled)
+    resource = dict_safe_get(response, ('resources', 0), return_type=dict, raise_return_type=False)
+
+    if not resource:
+        raise DemistoException('Unexpected response from CrowdStrike Falcon')
+
+    return resource
+
+
+def cs_falcon_ods_create_scan_command(args: dict) -> CommandResults:
+
+    resource = ods_create_scan(args, is_scheduled=False)
+    scan_id = resource.get('id')
+
+    if not scan_id:
+        raise DemistoException('Unexpected response from CrowdStrike Falcon')
+
+    query_scan_args = {
+        'ids': scan_id,
+        'wait_for_result': True,
+        'interval_in_seconds': args.get('interval_in_seconds'),
+        'timeout_in_seconds': args.get('timeout_in_seconds'),
+    }
+
+    return cs_falcon_ODS_query_scans_command(query_scan_args)
+
+
+def cs_falcon_ods_create_scheduled_scan_command(args: dict) -> CommandResults:
+
+    resource = ods_create_scan(args, is_scheduled=True)
+
+    human_readable = tableToMarkdown('Scheduled Scan Created', [resource.get("id")], headers=['Scan ID'])
```

Comment:
doesnt need to be returned as a table. Lets return a sentence like : Successfully created scheduled scan with id : id